website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,54 @@
+
+* Jest 27 will ship with a new test runner (`jest-circus`) and the node.js environment by default. `jest-jasmine2` and `jsdom` will still be bundled so users can keep using them by changing one-line each in the configuration.
+* Jest 28 will remove `jest-jasmine2` and `jsdom` from the default distribution of Jest. The packages will still be actively maintained as part of the Jest project and be published separately. Users will need to install these packages to use them.
+
+Upon upgrading to these major releases, Jest will guide you through the necessary changes. If you’d like to get ahead and migrate to the new architecture now you can upgrade to Jest 26 and add the following configuration options:
+
+```json
+{
+  "jest": {
+    "testEnvironment": "node",
+    "testRunner": "jest-circus/runner"
+  }
+}
+```
+
+Jest will continue to ship with `babel-jest` enabled by default. It currently powers a few Jest features beyond compiling modern JS syntax to something understood by current versions of Node.js and browsers. It also powers Jest's code coverage and mocking of ES modules. Jest currently ships with experimental support for V8 coverage and native support for ES Modules (more on that below!). It is not possible to mock static ES Modules per the specification without transforming the code first, so we will be encouraging patterns that work without the transformation we use Babel for today. Once V8 coverage and native ESM support stabilizes in Jest, we will also be able remove `babel-jest` as a default but we will keep maintaining it.
+
+## Other Breaking Changes in Jest 26
+
+* Dropped support for Node 8 ([#9423](https://github.com/facebook/jest/pull/9423))
+* `[jest-environment-jsdom]` Upgrade `jsdom` to v16 ([#9606](https://github.com/facebook/jest/pull/9606))
+* `[jest-runtime]` Remove long-deprecated `require.requireActual` and `require.requireMock` methods ([#9854](https://github.com/facebook/jest/pull/9854))
+* `[jest-haste-map]` Removed `providesModuleNodeModules` ([#8535](https://github.com/facebook/jest/pull/8535))
+* `[jest-circus]` Fail tests if a test takes a done callback and have return values ([#9129](https://github.com/facebook/jest/pull/9129))
+* `[jest-circus]` Throw a proper error if a test / hooks is defined asynchronously ([#8096](https://github.com/facebook/jest/pull/8096))
+* ***TODO***
+
+## Stay Safe
+
+We are all currently experiencing an unprecedented time of uncertainty. If you are struggling financially, we would like to use [Jest’s OpenCollective fund](https://opencollective.com/jest) to help new and existing contributors. We have an existing [list of issues with a bounty](https://github.com/facebook/jest/issues?q=is%3Aissue+is%3Aopen+bounty+label%3A%22Has+Bounty%22) but are open to offering a bounty on any of our current open issues - you can mention that this issue should have a bounty in the issue or contact [@cpojer via private message on Twitter](https://twitter.com/cpojer).
```

Comment:
```suggestion
We are all currently experiencing an unprecedented time of uncertainty. If you are struggling financially, we would like to use [Jest’s Open Collective fund](https://opencollective.com/jest) to help new and existing contributors. We place [bounties on some issues](https://github.com/facebook/jest/issues?q=is%3Aissue+is%3Aopen+bounty+label%3A%22Has+Bounty%22) and are open to offering a bounty on any of our current open issues - you can mention that an issue should have a bounty in the issue or contact [@cpojer via private message on Twitter](https://twitter.com/cpojer).
```
* OpenCollective => Open Collective (correct branding)
* and make the bounty list sound a bit less pretentious (but still open and inviting) given there's currently only one on there (and who knows, maybe a future blog reader will click it and be presented with zero)